tools/scripts/generate_constant_assets.py

```diff
@@ -57,15 +58,17 @@ def add_asset_initialization(self, var_name: str, identifier: str) -> str:
         self.id_to_variable[identifier] = var_name
         return generated_text
 
-    def add_ethtoken_initialization(self, var_name: str, address: str) -> str:
+    def add_ethtoken_initialization(self, var_name: str, identifier: str) -> str:
         generated_text = ''
-        token = self.globaldb.get_ethereum_token(address=address)
+        _, chain, address = identifier.split(':')
+        chain = int(chain.split('/')[0])
+        token = self.globaldb.get_evm_token(address=address, chain=ChainID(chain))
         var_swappedfor = 'None'
         if token.swapped_for:
             if token.swapped_for in self.id_to_variable:
                 var_swappedfor = self.id_to_variable[token.swapped_for]
             else:
-                var_swappedfor = f'{strethaddress_to_identifier(address).upper()}_swappedfor'
+                var_swappedfor = f'swappedfor_{address}'.upper()
```

Comment:
something may be wrong here as the swapped fors in the generated assets file did not seem okay for ETH/ETH etc.

Reply:
Yes, there was an error in the sql query. I'm not sure why it worked with the rest of the information. Fixed